lib/druid/filter.rb

```diff
@@ -299,7 +307,35 @@ def initialize(dimension, bounds)
       @bound = {
         type: 'radius',
         coords: bounds.first,
-        radius: bounds.last,
+        radius: bounds.last
+      }
+    end
+  end
+
+  class BoundFilter < Filter
+    include BooleanOperators
+
+    def initialize(dimension, params)
+      super()
+      @type = 'bound'
+      @dimension = dimension
+      @ordering = params[:ordering]
+      @upper = params[:upper]
+      @upperStrict = params[:upperStrict]
+    end
+  end
+
+  class SearchFilter < Filter
+    include BooleanOperators
+
+    def initialize(dimension, value)
+      super()
+      @type = 'search'
+      @dimension = dimension
+      @query = {
+        type: 'contains',
+        value: value,
+        caseSensitive: true
```

Comment:
is caseSensitive necessary here.

Reply:
ya, based on the druid query generated from pivot
<img width="599" alt="Screen Shot 2020-11-02 at 1 49 58 PM" src="https://user-images.githubusercontent.com/2039089/97887468-421d9900-1d2a-11eb-8a5d-74d0846a8110.png">

I have parameterized it, so it wont be true always.